packages/frontend/src/features/catalog/components/CatalogPanel.tsx

```diff
@@ -96,6 +100,20 @@ function getSelectionList(tree: CatalogTreeType): CatalogSelection[] {
     return selectionList;
 }
 
+enum FilterType {
+    Dimensions = 'dimensions',
+    Metrics = 'metrics',
+    HideBaseTables = 'hideBaseTables',
```

Comment:
maybe we can remove this until implemented?

Reply:
Sure!